supervisor/utils/__init__.py

```diff
@@ -39,20 +39,21 @@ async def wrap_api(api, *args, **kwargs):
     return wrap_api
 
 
-def check_port(address: IPv4Address, port: int) -> bool:
+async def check_port(
+    loop: asyncio.AbstractEventLoop, address: IPv4Address, port: int
```

Comment:
getting the loop is old school, you can call asyncio directly to get current loop back

Reply:
This came from the implementation proposed by bdraco in https://github.com/home-assistant/supervisor/pull/4678.

@bdraco any particular reason you added the loop as an argument?

Reply:
So reason is typically performance, however it got optimized e.g. in https://github.com/python/cpython/issues/100363.

I quickly checked, it is still measurable (`get_running_loop()` Mean +- std dev: 1.39 ms +- 0.02 ms`, property lookup Mean +- std dev: 297 us +- 6 us). But that is for 10000 calls, so rather negligible :sweat_smile:  

I'll go for `asyncio.get_running_loop()` to make the function simpler :+1: